static_frame/core/store.py

```diff
@@ -142,6 +168,13 @@ def label_decode(self, label: str) -> tp.Hashable:
             return self.label_decoder(label)
         return label
 
+    def to_store_config_he(self) -> 'StoreConfigHE':
+        '''
+        Return a ``StoreConfigHE`` version of this StoreConfig.
+        '''
+        return StoreConfigHE(**{attr: getattr(self, attr) for attr in self.__slots__})  #type: ignore [no-untyped-call]
```

Comment:
I think we should exclude the not hashable attrs here, rather than in the `StoreConfigHE`. We can define the not hashable in this class, too.